artmaker/src/commonMain/kotlin/io/fbada006/artmaker/customcolorpalette/SelectorIndicator.kt

```diff
@@ -22,6 +22,11 @@ import androidx.compose.ui.graphics.drawscope.DrawScope
 import androidx.compose.ui.graphics.drawscope.Stroke
 import androidx.compose.ui.unit.dp
 
+/**
+ * Vertically draw the Selector Indicator.
+ *
+ * @param amount Represents the value from which the Half Indicator Thickness will be subtracted from to get the Selector Indicator's [Offset].
```

Comment:
You will also notice, @emmanuelmuturia , that I have removed all the names of the methods included in the documentation of the method. That is a bit like defining a word using the word itself.